app/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,6 @@
     <string name="message_no_items_to_show">No items to show</string>
     <string name="no_description_available">No description available</string>
     <string name="img_content_description">Image content description</string>
+    <string name="CharacterDetailDialog">CharacterDetailDialog</string>
```

Comment:
I think it'd better to use the same naming_convention for all the strings in the project

Reply:
+1

Reply:
done